integration-tests/integration/coordinator/remove-statement.spec.js

```diff
@@ -0,0 +1,256 @@
+      prisoner().contains('Smith, Norman')
+      reporter().contains('James Stuart')
+      viewStatementsButton().click()
+    }
+
+    const viewStatementsPage = ViewStatementsPage.verifyOnPage()
+    viewStatementsPage.statements().then(result =>
+      expect(result).to.deep.equal([
+        { username: 'MRS_JONES name', badge: '', link: '', isOverdue: false, isUnverified: false },
+        {
+          username: 'TEST_USER name',
+          badge: 'REMOVAL REQUEST',
+          link: 'View removal request',
+          isOverdue: false,
+          isUnverified: false,
+        },
+      ])
+    )
+    viewStatementsPage.statementLink(1).click()
+
+    const viewRemovalRequestPage = ViewRemovalRequestPage.verifyOnPage()
+    viewRemovalRequestPage.name().contains('TEST_USER name')
+    viewRemovalRequestPage.userId().contains('TEST_USER')
+    viewRemovalRequestPage.location().contains('Moorland')
+    viewRemovalRequestPage.emailAddress().contains('TEST_USER@gov.uk')
+    viewRemovalRequestPage.removalReason().contains('not working')
+    viewRemovalRequestPage.confirm()
+    viewRemovalRequestPage.continue().click()
+
+    const confirmStatementDeletePage = ConfirmStatementDeletePage.verifyOnPage('TEST_USER name')
+    confirmStatementDeletePage.confirm()
+    confirmStatementDeletePage.continue().click()
+
+    viewStatementsPage
+      .statements()
+      .then(result =>
+        expect(result).to.deep.equal([
+          { username: 'MRS_JONES name', badge: '', link: '', isOverdue: false, isUnverified: false },
+        ])
+      )
+  })
+
+  it(`A coordinator can refuse a statment removal request`, () => {
+    cy.task('stubCoordinatorLogin')
+    cy.login()
+
+    seedReport()
+
+    const notCompletedIncidentsPage = NotCompletedIncidentsPage.goTo()
+    notCompletedIncidentsPage.getTodoRows().should('have.length', 1)
+
+    {
+      const { prisoner, reporter, viewStatementsButton } = notCompletedIncidentsPage.getTodoRow(0)
+      prisoner().contains('Smith, Norman')
+      reporter().contains('James Stuart')
+      viewStatementsButton().click()
+    }
+
+    const viewStatementsPage = ViewStatementsPage.verifyOnPage()
+    viewStatementsPage.statements().then(result =>
+      expect(result).to.deep.equal([
+        { username: 'MRS_JONES name', badge: '', link: '', isOverdue: false, isUnverified: false },
+        {
+          username: 'TEST_USER name',
+          badge: 'REMOVAL REQUEST',
+          link: 'View removal request',
+          isOverdue: false,
+          isUnverified: false,
+        },
+      ])
+    )
+    viewStatementsPage.statementLink(1).click()
+
+    const viewRemovalRequestPage = ViewRemovalRequestPage.verifyOnPage()
+    viewRemovalRequestPage.name().contains('TEST_USER name')
+    viewRemovalRequestPage.userId().contains('TEST_USER')
+    viewRemovalRequestPage.location().contains('Moorland')
+    viewRemovalRequestPage.emailAddress().contains('TEST_USER@gov.uk')
```

Comment:
meh.. don't really need to test the same thing in each test - just making sure it's fine in one test is probably enough